src/3rd_party_add.c

```diff
@@ -94,18 +94,11 @@ static int remove_repo(const char *repo_name)
 
 static bool confirm_certificate(const char *cert)
 {
-	int c;
-
 	info("Importing 3rd-party repository public certificate:\n\n");
 	signature_print_info(cert);
 
 	info("\n");
-	info("Do you want to accept this certificate? (y/N): ");
```

Comment:
I think it's important to print a message asking if we want to accept the certificate. It's not clear to me just to say "Do you want to continue?"

Reply:
how about now?